src/Database/Schema/Column.php

```diff
@@ -42,26 +43,28 @@ class Column
      * @param string|null $after Name of the column to add this column after
      * @param string|null $onUpdate MySQL 'ON UPDATE' function
      * @param string|null $comment Comment for the column
-     * @param bool $unsigned Whether the column is unsigned
+     * @param bool|null $unsigned Whether the column is unsigned
      * @param string|null $collate Collation for the column
      * @param int|null $srid SRID for geometry fields
+     * @param string|null $baseType The basic schema type if the column type is a complex/custom type.
      */
     public function __construct(
         protected string $name,
         protected string $type,
-        protected bool $null = true,
+        protected ?bool $null = null,
         protected mixed $default = null,
         protected ?int $length = null,
         protected bool $identity = false,
-        protected ?string $generated = PostgresSchemaDialect::GENERATED_BY_DEFAULT,
+        protected ?string $generated = null,
         protected ?int $precision = null,
         protected ?int $increment = null,
         protected ?string $after = null,
         protected ?string $onUpdate = null,
         protected ?string $comment = null,
-        protected bool $unsigned = true,
+        protected ?bool $unsigned = null,
```

Comment:
Do we need to validate this is set for types that support it?

Reply:
We could. I initially had this defaulted to false, but it caused a bunch of regressions in tests, so I removed it.